### DISCUSSION TITLE
Update the Curios Replace command to allow entity and block selectors

### DISCUSSION THREAD

**Discussion by SumaniPardia** [TheIllusiveC4/Curios#324] [Ideas]
Would it be possible to add to the "/curios replace" command so in addition to "with" it accepts "from" similar to the Minecraft "/item replace" command?  This will add functionality such as allowing datapacks to store a players current curios in a chest (already doable) and then later remove them from the chest and place them back into the players curios slots (not currently doable).

i.e.
`/curios replace @s back 2 from block 250 12 364 container.0`

**Comment by TheIllusiveC4** [comment]
Probably? What Minecraft version are you on?

**Comment by SumaniPardia** [comment]
I'm playing on 1.19.2, thanks.
